src/extensions/updateHelper.ts

```diff
@@ -279,23 +254,12 @@ export async function updateToVersionFromPublisherSource(
     );
   }
   let registryEntry;
-  let sourceOrigin;
   try {
-    // Double check that both publisher and extension ID match
-    // If the publisher and extension ID both match, we know it's an official extension (i.e. it's specifically listed in our Registry File)
-    // Otherwise, it's simply a published extension in the Registry
     registryEntry = await resolveSource.resolveRegistryEntry(existingSpec.name);
-    sourceOrigin =
-      registryEntry.publisher === refObj.publisherId
-        ? SourceOrigin.OFFICIAL_EXTENSION
-        : SourceOrigin.PUBLISHED_EXTENSION;
   } catch (err) {
-    sourceOrigin = SourceOrigin.PUBLISHED_EXTENSION;
+    // Ignore errors if we can't fetch the registry
```

Comment:
is this registry.json?

silently failing worries me... do we have some kind of error reporting for the CLI?

Reply:
Adding a debug log here, but I think this is necessary to ignore. We want to be able to take down the registry at some point without breaking this. In the case of an outage before then, this would just accidentally allow updates below minVersion, which is a small corner case that we're already in the process of migrating to the backend.

Reply:
sg with noop just wanted to get some logs here in case we need to figure out how often this happens